tests/integration/it_tables_test.py

```diff
@@ -1,30 +1,89 @@
 """
-@todo #169:30min Write test for creating and editing Table model. To test
- it correctly it's needed to create relations like `TableShape` and so on
- in tests. There is`factory-boy` library in requirements added. It helps
- to create instance with its relations automatically, look at
- `tests/factories.py` and see
- https://factoryboy.readthedocs.io/en/latest/orms.html#sqlalchemy. Implement
- these factories for related models and then write tests.
+@todo #234:30min Timeless ModelForm is not working properly, the
+ Table form in test_create cannot be saved. The problem is that it
+ does not pass form data parameters to model constructor properly.
+ Parameters are passed through *args but ModelForm works uses **kwargs
+ to instantiate model. Once this is fixed, remove skip annotations
+ from tests.
 """
 import pytest
+
 from http import HTTPStatus
 
+from flask import url_for
+
+from timeless.restaurants.models import Table
+
```

Comment:
`http` is python native lib.
`pytest, flask` are thirdparty
now it should be in this way:
```
from http import HTTPStatus

import pytest
from flask import url_for

from timeless.restaurants.models import Table
```